src/perf/bin/drvmain.cpp

```diff
@@ -0,0 +1,604 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Kernel Mode Performance Driver
+
+--*/
+
+#include <quic_platform.h>
+#include "msquic.h"
+#include <quic_driver_main.h>
+#include "perfioctls.h"
+
+#include "quic_trace.h"
+#ifdef QUIC_CLOG
+#include "driver.cpp.clog.h"
+#endif
+
+#define QUIC_PERF_TAG 'frPQ' // QPrf
+
+DECLARE_CONST_UNICODE_STRING(QuicPerfCtlDeviceName, L"\\Device\\quicperformance");
+DECLARE_CONST_UNICODE_STRING(QuicPerfCtlDeviceSymLink, L"\\DosDevices\\quicperformance");
+
+typedef struct QUIC_DEVICE_EXTENSION {
+    EX_PUSH_LOCK Lock;
+
+    _Guarded_by_(Lock)
+        LIST_ENTRY ClientList;
```

Comment:
style: don't tab over LIST_ENTRY ClientList. Match the rest of the variables.

Reply:
This will need to be fixed in the BVTs then at some point too.